specification/containerregistry/resource-manager/Microsoft.ContainerRegistry/preview/2022-02-01-preview/containerregistry.json

```diff
@@ -15,6 +15,259 @@
+            }
+          }
+        },
+        "x-ms-examples": {
+          "CacheRuleCreate": {
+            "$ref": "./examples/CacheRuleCreate.json"
+          }
+        },
+        "x-ms-long-running-operation": true
+      },
+      "delete": {
+        "tags": [
+          "CacheRules"
+        ],
+        "summary": "Deletes a cache rule resource from a container registry.",
+        "operationId": "CacheRules_Delete",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupParameter"
+          },
+          {
+            "$ref": "#/parameters/RegistryNameParameter"
+          },
+          {
+            "$ref": "#/parameters/CacheRuleNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The request was successful; the request was well-formed and received properly."
+          },
+          "202": {
+            "description": "The request was successful; the operation will complete asynchronously."
+          },
+          "204": {
+            "description": "The cache rule does not exist in the subscription."
+          },
+          "default": {
+            "description": "Error response describing why the operation failed. If the resource does not exist, 404 (Not found) is returned.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "CacheRuleDelete": {
+            "$ref": "./examples/CacheRuleDelete.json"
+          }
+        },
+        "x-ms-long-running-operation": true
+      },
+      "patch": {
+        "tags": [
+          "CacheRules"
+        ],
+        "summary": "Updates a cache rule with the specified parameters.",
+        "operationId": "CacheRules_Update",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupParameter"
+          },
+          {
+            "$ref": "#/parameters/RegistryNameParameter"
+          },
+          {
+            "$ref": "#/parameters/CacheRuleNameParameter"
+          },
+          {
+            "name": "cacheRuleUpdateParameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/CacheRuleUpdateParameters"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The request was successful; the request was well-formed and received properly.",
+            "schema": {
+              "$ref": "#/definitions/CacheRule"
+            }
+          },
+          "201": {
```

Comment:
>201 [](http://example.com/codeflow?start=11&length=3)

Pl address the errors that show up in the linter validation pipeline

[https://github.com/Azure/azure-rest-api-specs/pull/21496/checks?check_run_id=9395186769](https://github.com/Azure/azure-rest-api-specs/pull/21496/checks?check_run_id=9395186769)